### DISCUSSION TITLE
github logout

### DISCUSSION THREAD

**Discussion by smeriweather** [atom/atom#22589] [Q&A]
How do I get atom to logout of github and login with different credentials

**Comment by DeeDeeG** [comment]
On Windows, you can go into the Credential Manager and delete your GitHub login credentials.

- https://support.microsoft.com/windows/accessing-credential-manager-1b5c916a-6a16-889f-8581-fc16e8165ac0

On macOS, it is similar, but the app is called Keychain Access:

- https://support.apple.com/guide/keychain-access/what-is-keychain-access-kyca1083/mac

On Ubuntu Linux, try SeaHorse or KWalletManager

- https://wiki.gnome.org/Apps/Seahorse
- https://apps.kde.org/kwalletmanager5/

Once the credential is deleted, you can restart Atom, and the GitHub tab will ask again for you to log in. Or instead of deleting the credential, you can make a personal access token (https://docs.github.com/en/github/authenticating-to-github/keeping-your-account-and-data-secure/creating-a-personal-access-token) (should have `read:org`, `user:email`,  `repo` and optionally `workflow` permissions) and use that in place of your previous token.
